### DISCUSSION TITLE
Quetion about InterfaceWithPython

### DISCUSSION THREAD

**Discussion by peterrjohnston** [SCIInstitute/SCIRun#2319] [Q&A]
Hello,

I have a python script that I am trying to run, but it just sits there spinning its little wheel. The IWP script is supposed to call some other python code that I have written and is saved in 'my_code.py'. The script seems to get stuck at the line

from my_code import *         (this line works fine in Ipython)

The line immediately preceding this is a sys.path.append command with the argument being the directory containing 'my_code.py'.

Clearly, I am missing something here. Any thoughts would be much appreciated. I can provide further details if necessary.

Thanks very much,

Peter.

**Comment by jab0707** [comment]
Just to be clear your code does something like this right?

```Python
import sys
sys.path.append(pathToMy_code)
from my_code import *
```
If so, that should work. You could double-check where things are getting stuck by adding a print statement at the top of my_code to see how far into hat it is getting. What does my_code do? There might be something in there that it is hanging on.
